lib/query/compiler.js

```diff
@@ -396,6 +397,20 @@ assign(QueryCompiler.prototype, {
     return sql.length > 1 ? sql.join(' ') : '';
   },
 
+  // Compiles all `raw` statements on the query.
+  raws() {
```

Comment:
At some point there was meant to be a way to pass `?` to raw SQL string without need to escape it in case if no bindings were passed to the `.raw` call. So just concatenating SQL and bindings would break that feature.

Though I'm not sure if that was ever implemented.